app/src/views/team/[id]/VestingView.vue

```diff
@@ -1,175 +1,57 @@
 <template>
-  <VestingStats :vestings="vestings" :symbol="tokenSymbol ? tokenSymbol : ''" />
+  <VestingStats :reloadKey="reloadKey" />
   <div>
-    <CardComponent title="Vesting Overview">
-      <template #card-action>
-        <ButtonUI
-          size="sm"
-          data-test="toggle-vesting-view"
-          :variant="displayActive ? 'secondary' : 'ghost'"
-          class="w-max"
-          @click="displayActive = !displayActive"
-        >
-          <IconifyIcon
-            :icon="displayActive ? 'heroicons-outline:inbox' : 'heroicons-outline:archive-box'"
-            class="size-6"
-          />{{ displayActive ? 'actives' : 'archived' }}
-        </ButtonUI>
-        <ButtonUI
-          v-if="team?.ownerAddress == useUserDataStore().address"
-          size="sm"
-          variant="primary"
-          class="w-max"
-          @click="addVestingModal = true"
-          data-test="createAddVesting"
-        >
-          <IconifyIcon icon="heroicons-outline:plus-circle" class="size-6" /> add vesting
-        </ButtonUI>
-        <ModalComponent v-model="addVestingModal">
-          <CreateVesting
-            v-if="team?.id"
-            :teamId="Number(team.id)"
-            :tokenAddress="sherToken?.address ?? ''"
-            @closeAddVestingModal="addVestingModal = false"
-            @reloadVestingInfos="getVestingInfos"
-            :vestings="activeVestings"
-          />
-        </ModalComponent>
-      </template>
-      <VestingFlow :vestings="vestings" @reloadVestingInfos="getVestingInfos" />
-    </CardComponent>
+    <ButtonUI
+      v-if="team?.ownerAddress == userAddress"
+      size="sm"
+      variant="primary"
+      class="w-max"
+      @click="addVestingModal = true"
+      data-test="createAddVesting"
+    >
+      <IconifyIcon icon="heroicons-outline:plus-circle" class="size-6" /> add vesting
+    </ButtonUI>
```

Comment:
You can add the button and the modal, in a component, and use the component in vesting overview action slot